lib/enums/incident_type.dart

```diff
@@ -0,0 +1 @@
+enum IncidentType { maintenance, scheduled, unresolved, upcoming }
```

Comment:
Can we add a newline here?

Reply:
Sure, no problem. Would you like me to create a new commit so that you can squash them together? Or would you prefer that I do a rebase and create one new commit?